docs/en/latest/xrpc/redis.md

```diff
@@ -0,0 +1,102 @@
+|----------------------------------------------|---------------|----------|-----------------------------------------------|--------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| faults | array[object]        | False    |                                               |  | Fault injections which can be applied based on the commands and keys |
+
+Fields under an entry of `faults`:
+
+| Name | Type          | Required | Default                                       | Valid values                                                       | Description                                                                                                                                                                                                                                           |
+|----------------------------------------------|---------------|----------|-----------------------------------------------|--------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| commands | array[string]        | True    |                                               | ["get", "mget"]  | Commands fault is restricted to |
+| key | string        | False    |                                               | "blahblah"  | Key fault is restricted to |
+| delay | number        | True    |                                               | 0.1  | Duration of the delay in seconds |
+
+## Example usage
+
+Assumed the APISIX is proxying TCP on port 9101, and the Redis is listening on port 6379.
+
+Let's create a stream route:
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/stream_routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+{
+    "upstream": {
+        "type": "none",
+        "nodes": {
+            "127.0.0.1:6379": 1
+        }
+    },
+    "protocol": {
+        "name": "redis",
+        "conf": {
+            "faults": [{
+                "commands": ["get", "ping"],
+                "delay": 5
+            }]
+        }
+    }
+}
+'
+```
+
+Once you have configured the stream route, as shown above, you can make a request to it:
+
+```shell
+$ redis-cli -p 9101
```

Comment:
```suggestion
redis-cli -p 9101
```

Reply:
Better to keep the prompt, so that it can be distinguish from the cmd's output

Reply:
Why not split the command and output into two code blocks? Because that's what our other docs do. https://apisix.apache.org/docs/apisix/next/plugins/redirect#example-usage